packages/babel-parser/test/fixtures/es2015/uncategorised/223/output.json

```diff
@@ -1,32 +1,32 @@
 {
   "type": "File",
   "start": 0,
-  "end": 22,
+  "end": 20,
```

Comment:
This is because https://github.com/babel/babel/blame/master/packages/babel-parser/test/fixtures/es2015/uncategorised/223/input.js changed recently